parser/src/main/java/io/specmesh/apiparser/model/KafkaBinding.java

```diff
@@ -35,26 +37,28 @@
  */
 
 /** Pojo representing a Kafka binding */
-@Value
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Don't think these are needed for this PR. Not having them reduces the size of your API, which is a good thing.

Also, consider if this API lets the user build specs that are invalid.  This should be avoided. (I think lombok supports _required_ properties, which you should set)